js/ps5/addons/p5.sound.min.js

```diff
@@ -1,3 +0,0 @@
-/** [p5.sound]  Version: 1.0.1 - 2021-05-25 */ 
```

Comment:
having both the minified and the non-minified version of the p5.sound files makes no sense. Functionally, they are literally the same file, only difference is that one is unreadable - without spaces and indentations and with short variable names for everything etc.